packages/merge-on-green/test/Fixtures/events/longListOfCommits.json

```diff
@@ -0,0 +1,105 @@
+[{"sha": "6dcb09b5b57875f334f61aebed695e2e4193db5e"},
```

Comment:
It certainly works, but can it be dynamically created by a for loop so that it's more expressive?